CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -40,6 +40,25 @@ public WpfImeKeyboardHandler(ChromiumWebBrowser owner) : base(owner)
         {
         }
 
+        /// <summary>
+        /// Get the outermost element of the browser 
+        /// </summary>
+        /// <param name="control">The browser</param>
+        /// <returns>The outermost element </returns>
+        private FrameworkElement GetOutermostElement(FrameworkElement control)
```

Comment:
Please move private method to bottom of class.

Reply:
Got it, modified.